app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -43,7 +50,8 @@ def index
     param :organization_id, :number, :desc => N_("Organization identifier")
     param :id, :number, :desc => N_("Subscription identifier"), :required => true
     def show
-      respond :resource => @subscription
+      @resource = Katello::Pool.with_identifiers(params[:id]).first
+      respond(@resource)
     end
 
     def available
```

Comment:
this is what available_for is replacing, I believe it is subscriptions/available with content host or activation key id passed in. I can probably remove this function, but I wasn't sure if that would be messing up anything for hammer?